.golangci-sample.yml

```
@@ -0,0 +1,151 @@
+    - typecheck
+    - unused
+    
+    # Code quality
+    - gofmt
+    - goimports
+    - misspell
+    - unconvert
+    - gocyclo
+    - gosec
+
+issues:
```

YAMLlint (1.37.1)

[error] duplication of key "issues" in mapping

(key-duplicates)